generator/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+repositories {
+    mavenLocal()
+    jcenter()
+    mirego 'common-web'
+}
+
+dependencies {
+    compile project(':shared')
+    compile files("${System.properties['java.home']}/../lib/tools.jar")
+    compile 'com.google.guava:guava:18.0'
+    compile 'com.google.javascript:closure-compiler:v20141215'
+    compile 'org.mozilla:rhino:1.7R4'
+    compile 'com.google.code.findbugs:annotations:2.0.0'
+
+    compile 'junit:junit:4.8.1'
+    //testCompile project(':server')
```

Comment:
I don't know... It was in pom.xml, I hadn't migrated server yet, comment to test and it worked.